mdpow/fep.py

```diff
@@ -127,11 +127,13 @@
 import copy
 from subprocess import call
 import warnings
+import sys
```

Comment:
remove, use try/except below

Reply:
Done in recent commit

Reply:
It's still here – did you not push the commit?

Reply:
```suggestion
```